Packs/IntegrationsAndIncidentsHealthCheck/Scripts/GetFailedTasks/GetFailedTasks.py

```diff
@@ -28,14 +59,20 @@ def main():
             break
 
     for incident in total_incidents:
+        if tenant_name:
+            uri = f'acc_{tenant_name}/investigation/{str(incident["id"])}/workplan/tasks'
+        else:
+            uri = f'investigation/{str(incident["id"])}/workplan/tasks'
```

Comment:
Instead of `tenant_name`, you can use `#/`.
You can see a similar thing here: https://github.com/demisto/content/blob/master/Packs/CommonScripts/Scripts/FeedRelatedIndicatorsWidget/FeedRelatedIndicatorsWidget.py#L51